contribute/style-guides/terminology.md

```diff
@@ -0,0 +1,15 @@
+# Terminology used throughout Grafana code base
+
+This document aims to define and standardize terminology used throughout Grafana source code.
+
+## TLS/SSL
+
+The acronyms [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) (Transport Layer Security and 
```

Comment:
I recommend that we use a definition list of some sort rather than paragraph style. Can you test this format out:

```
kramdown
: A Markdown-superset converter
```

Reply:
@osg-grafana I'm not familiar with using kramdown, where do you want this to be inserted?

Reply:
This is going to need more input from @grafana/docs-squad. As simple as this PR sounds, we are laying some groundwork here in terms of a model that others can contribute to, so please bear with us as we come to consensus. We have a meeting today where we can discuss this.

Reply:
I included this particular markdown style in the code changes in the other parts of the doc. Based on what we do here, we could stand to update the Markdown guide (if we go with Markdown for this doc) to include definition list style at https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.md#lists.

Reply:
Is kramdown supported by the Blackfriday engine? That's what we use according to [Contributing to docs](https://github.com/grafana/grafana/blob/master/contribute/documentation.md#style-and-formatting).

Reply:
I'm still not sure exactly what kind of changes you have mind. I don't think it's worth blocking this PR until a style is decided on though. It's probably best to agree on the basics, and experiment with style independently.

Reply:
Agreed.